modules/nodejs/manifests/init.pp

```diff
@@ -1,9 +1,15 @@
 class nodejs (
   $path = "/vagrant/extensions/nodejs"
 ) {
-  package { 'clang-3.4':
-    ensure => latest
+  if $lsbdistcodename == 'xenial' {
```

Comment:
I think this is a legacy fact will `$lsb_release` work here instead? If not I'm happy to merge this!